languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/disambiguation.xml

```diff
@@ -2172,6 +2172,21 @@ Copyright © 2013 Markus Brenneis, Daniel Naber, Jan Schreiber
     <!-- ============================================================================================================== -->
     <!-- IGNORE SPELLING -->
 
+    <rule name="auf-... und zumachen etc." id="VERBPRÄFIX_HYPHEN">
+        <!--Wir müssen los- und fortrennen.-->
+        <!--Das Los- und Fortrennen ist keine gute Idee.-->
+        <!--Das ist für die Be- und Entladung.-->
+        <pattern>
+            <marker>
+                <token regexp="yes">(ge|fort|bei?|a[bn]|au[fs]|e(in|nt|r)|hin(ein|aus)?|her(ein|aus)?|los|mi(t|ss)|vor(wärts)?|weg|zu(rück)?|r(aus|ein)|ver|zer|um|über|unter|durch)-</token>
```

Comment:
Die Regex in den beiden Tokens sind fast, aber nicht ganz gleich. Ist das gewollt? Wenn sie gleich sein sollten wäre es vielleicht besser, ein Entitiy (`&...;`) zu definieren, um Redundanzen zu vermeiden.

Reply:
https://github.com/languagetool-org/languagetool/pull/9780/commits/9eff3e355e28cf628bbd857300cf08b7a2682b83